evaluation/degradation_eval.py

```diff
@@ -1,62 +1,77 @@
 import pathlib
 import random
 import csv
+import simpleaudio as sa
```

Comment:
```suggestion
from simpleaudio import WaveObject
```

Nur als Idee, dann liest sich der Aufruf in Zeile 45 schöner 🙂